signups/admin.py

```diff
@@ -23,3 +26,17 @@ class SignupTargetAdmin(admin.ModelAdmin):
     fields = ('name', 'identifier')
     prepopulated_fields = {'identifier': ('name',)}
     inlines = (SignupInline,)
+    actions = ["export_participants"]
+
+    def export_participants(self, request, queryset):
+        if queryset.count() > 1:
+            return self.message_user(request, _(u"Valitse vain yksi Signup Target"), messages.ERROR)
+
+        signup_target = queryset.first()
+        participants = signup_target.users.all()
+
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename=orders.csv'
+        export_participants_as_csv(participants, response)
+        return response
+    export_participants.short_description = _("Vie osallistujen lista CSV-tiedostoon")
```

Comment:
Should be `osallistujien`. Actually maybe "Lataa ilmoittautuneiden lista CSV-tiedostona." would be better.